devops/scripts/clean_ci_validation_envs.sh

```diff
@@ -65,14 +65,16 @@ while read -r rg_name rg_ref_name; do
   fi
 done
 
-# check if any workflows run on the main branch
+# check if any workflows run on the main branch (except the cleanup one)
 # to prevent us deleting a workspace for which an E2E (on main) is currently running
-if [ $(gh api "https://api.github.com/repos/microsoft/AzureTRE/actions/runs?branch=main&status=in_progress" --jq ".total_count") == 0 ]
+if [[ -z $(gh api "https://api.github.com/repos/microsoft/AzureTRE/actions/runs?branch=main&status=in_progress" | jq '.workflow_runs | select(.[].name != "Clean Validation Environments")') ]]
```

Comment:
is there a possibility to use id of some kind instead of name? so it won't break if someone rename

Reply:
I used the name from the github context which is even better...